lib/relation-definition.js

```diff
@@ -2045,7 +2045,11 @@ RelationDefinition.embedsOne = function(modelFrom, modelToRef, params) {
     methods: params.methods,
   });
 
-  var opts = {type: modelTo};
+  var opts = Object.assign(
+    {},
+    params.options && params.options.property ? params.options.property : {},
```

Comment:
A personal opinion: naming the property mapping configuration as `property` sounds a little ambiguous when I read the code, how about `propertyMapping` or `propertyConfig`?

Reply:
I can see how it sounds ambiguous but on the other hand isn't `propertyMapping` and `propertyConfig` a bit redundant? Since you're effectively saying `options.propertyOptions` in that context?